setup.py

```diff
@@ -100,7 +100,7 @@ class BDistWheel(CompileCatalogMixin, BaseBDistWheel):
     setup_requires=["Babel>=2.6.0"],
     install_requires=["MarkupSafe"],
     extras_require={
-        "ipaddress": ['ipaddress;python_version<="2.7"'],
+        "ipaddress": ['ipaddress;python_version<"3"'],
```

Comment:
Uhh, not to drag this out too long, but technically this should be "<3.3". :sweat: Then again, we don't support <3.4, so... :man_shrugging:

Reply:
Updated with 325367f6469dce804f73b7891b0f64254ac37ec1. 👍

Reply:
Not sure if you've seen the latest push, and don't want to merge without full review.